deps.ffmpeg/30-svt-av1.zsh

```diff
@@ -111,12 +111,11 @@ fixup() {
       ;;
     windows-x*)
       if (( shared_libs )) {
-        mv (#i)"${target_config[output_dir]}"/lib/libsvtav1*.dll ${target_config[output_dir]}/bin/
```

Comment:
On second look this change needs to be reverted - the `rm` command further below relies on the DLL to have an all-lowercase name, I also faintly remember other issues raised from the file names being mixed case.

Reply:
Also the DLL still needs to be moved to the `bin` directory.

Reply:
I have reverted this commit